libres/lib/enkf/enkf_plot_data.cpp

```diff
@@ -116,28 +114,20 @@ void enkf_plot_data_load(enkf_plot_data_type *plot_data, enkf_fs_type *fs,
         mask = bool_vector_alloc_copy(input_mask);
     else
         mask = bool_vector_alloc(ens_size, false);
-    state_map_select_matching(state_map, mask, STATE_HAS_DATA, true);
 
+    state_map_select_matching(state_map, mask, STATE_HAS_DATA, true);
     enkf_plot_data_resize(plot_data, ens_size);
     enkf_plot_data_reset(plot_data);
+
     {
-        const int num_cpu = 4;
-        thread_pool_type *tp = thread_pool_alloc(num_cpu, true);
         for (int iens = 0; iens < ens_size; iens++) {
             if (bool_vector_iget(mask, iens)) {
                 enkf_plot_tvector_type *vector =
                     enkf_plot_data_iget(plot_data, iens);
-                arg_pack_type *work_arg = plot_data->work_arg[iens];
```

Comment:
Should work_arg now be removed as its not used?

Reply:
Good point...  I didn't track it outside of `enkf_plot_data_load()` but it is actually not used nor referenced anywhere outside the file `enkf_plot_data.cpp`